atef/widgets/config/window.py

```diff
@@ -491,7 +507,7 @@ def __init__(
         self,
         *args,
         config_file: ConfigurationFile,
-        full_path: str,
+        full_path: Optional[str] = None,
```

Comment:
This was used in a previous version of the PR (the 5 minute version) but I could revert it now if it's distasteful. The widget works properly when full_path is given as None.